### ISSUE TITLE
Progress bar on move/delete/copy

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/android#237]
When deleting/moving/copying files there should be shown what is going on, e.g. deleting 1 of 10 filename.
@jancborchardt I know you wanted non blocking notifications. But what if some actions block other actions: deleting 100 files and then immediately try to share one of this file?

**Comment by jancborchardt** [comment]
What we should do for example in the case of deletion is immediately vanish the files locally. Sure it could take longer to be deleted on the server, but that’s nothing we should need to wait for. Only if there’s an error.

That also takes care of your case 
> deleting 100 files and then immediately try to share one of this file

cause they are simply not shown anymore.

**Comment by mario** [comment]
I agree @jancborchardt, but will take a bit of time to do proper planning/design :)
